src/composables/useDevices.js

```diff
@@ -106,6 +108,15 @@ export function useDevices(video, initializeOnMounted) {
 
 		virtualBackground.value.connectTrackSink('default', videoTrackToStream.value, 'video')
 
+		const blurBackgroundEnabled = getTalkConfig('local', 'call', 'blur-background')
```

Comment:
should check store value here

Reply:
I will only use talk-config without any stored value